packages/shared/src/components/drawers/NavDrawer.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { ReactElement } from 'react';
+import {
+  Drawer,
+  DrawerPosition,
+  DrawerRef,
+  DrawerWrapperProps,
+} from './Drawer';
+import { NavDrawerItem, NavItemProps } from './NavDrawerItem';
+import { Button, ButtonSize, ButtonVariant } from '../buttons/Button';
+import { ArrowIcon } from '../icons';
+import classed from '../../lib/classed';
+
+interface NavDrawerProps {
+  drawerProps: Omit<DrawerWrapperProps, 'children'>;
+  items: NavItemProps[];
+  header?: string;
+}
+
+const NavDrawerHeader = classed(
+  'div',
+  'flex items-center gap-3 px-4 border-b border-theme-divider-tertiary h-12',
+);
+
+const NavDrawerContent = classed('div', 'flex flex-col px-4 py-5');
+
+const NavHeading = classed('h2', 'typo-body font-bold');
+
+export function NavDrawer({
+  drawerProps,
+  header,
+  items,
+}: NavDrawerProps): ReactElement {
+  const ref = React.useRef<DrawerRef>();
+  const {
+    position,
+    displayCloseButton,
+    closeOnOutsideClick,
+    ...otherDrawerProps
+  } = drawerProps;
+
+  return (
+    <Drawer
+      {...otherDrawerProps}
+      closeOnOutsideClick={false}
+      displayCloseButton={displayCloseButton}
+      position={position ?? DrawerPosition.Left}
+      ref={ref}
+      role="menu"
+      className={{
+        drawer: 'bottom-0 top-0 !max-h-none !rounded-none',
+        overlay: 'bg-transparent',
```

Comment:
Is it required to have a transparent bg? Usually when we open popups, etc. we add overlay so the users can focus on the shown display.

Reply:
Since in this case the drawer is full-screen, it does not make sense to show the overlay. This is just poor mans (me) implementation to hide the overlay without killing all the existing styling, e.g. the animation. 

On top of that, the animation looks rather ugly, if we put a blue overlay there and then in 300ms completely cover it.

Reply:
I switched to adding an `isFullScreen` prop to the `Drawer` itself. Now we only set the background if the drawer is not fullscreen, otherwise it's whatever is the default background.